modules/fedimint-mint-client/src/lib.rs

```diff
@@ -200,6 +208,34 @@ impl MintClientExt for Client {
             .await
     }
 
+    async fn await_mint_change(
+        &self,
+        operation_id: OperationId,
+        txid: TransactionId,
+    ) -> anyhow::Result<()> {
+        let tx_outcome = self.api().await_tx_outcome(&txid).await?;
+        if let TransactionStatus::Accepted {
+            epoch: _epoch,
+            outputs,
+        } = tx_outcome
+        {
+            // Assume that if the transaction has more than one output, the output
+            // at index 1 is the change.
+            if outputs.len() > 1 {
+                let (_, mint_client) = mint_client(self);
+                let out_point = OutPoint { txid, out_idx: 1 };
```

Comment:
Not sure if this is the most elegant way of awaiting the change from the mint module. It assumes something about the transaction structure. Open to suggestions!

Reply:
You could pass in an `Outpoint` instead of a `TransactionId`? Then at least you could move the assumption from this utility function to the caller?